plugins/katello/3.18/upgrade/index.md

```diff
@@ -78,6 +78,16 @@ If the above steps failed, please review /var/log/foreman-installer/katello.log
 
 Katello 3.18 supports migrations of Content to Pulp 3 for container, yum, and file content.  Depending on how much content is present, this can take a very long time.  The largest part of the process can be run without downtime.  This full process will need to be complete before upgrading to Katello 4.0.
 
+### File system requirements
+
+Upgrading to pulp3 does require some additional space, although the bulk of the content is not duplicated.  Ensure you have:
+
+1.  80% of the space /var/lib/mongodb/ consumes should be available in /var/opt/rh/rh-postgresql12/lib/pgsql/.  (If /var/lib/mongodb is 100GB, then 80GB needs to be free for postgresql).
```

Comment:
I'd reword point 1 to sound more like point 2.  Point 1 sounds a little funny following "Ensure that you have ..."